elements/rh-pagination/demo/vue.html

```diff
@@ -0,0 +1,61 @@
+<script type="module">
+  import '@rhds/elements/rh-pagination/rh-pagination.js';
+</script>
+
+<script src="https://cdn.jsdelivr.net/npm/vue@3.5.22/dist/vue.global.js"></script>
+
+<div id="app">
+  <rh-pagination>
+    <ol>
+      <li v-for="pageNum in pages" :key="pageNum">
+        <a :href="`#${pageNum}`" :aria-current="pageNum == currentPage ? 'page' : null" v-text="pageNum"></a>
+      </li>
+    </ol>
+  </rh-pagination>
+
+  <span v-text="message"></span> <span v-text="currentPage"></span>
+</div>
+
+<script>
+  // eslint-disable-next-line no-undef
+  const { createApp } = Vue;
```

Comment:
where is this loaded from? can you add it to the import map instead and explicitly import it here (which would also clear the lint error)?

Reply:
fixed in [7308bff](https://github.com/RedHat-UX/red-hat-design-system/pull/2636/commits/7308bff8a722634ee01170c535bf361fb2b04e88)